sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -106,10 +121,17 @@ def process(self, row):
     #                     'field1',
     #                     'value1',
     #                     timestamp=datetime.datetime.now())
-    self.batcher.mutate(row)
+    try:
+      self.batcher.mutate(row)
+    except MaxMutationsError:
+      self.service_call_metric.call(MaxMutationsError)
```

Comment:
You can't pass in an arbitrary exception to service_call_metric.call()
And in this case you are actually passing in the class instance instead of the object instance of the exception

Please pass in a canonical GCP status string
i.e. "ok", "not_found", unauthenticated, etc.

If that's not available a canonical http status code number can be passed in and mapped properly to the string.
i.e. 400, 401
https://cloud.google.com/apis/design/errors#grpc_mapping